.codecov.yml

```diff
@@ -0,0 +1,12 @@
+codecov:
+  coverage:
+    status:
+      project: 
+        enabled: true
+        target: auto
+        threshold: 100%
+      patch:
+        enabled: false
+      change:
+        enabled: false
+  comment: #ToDO - Disable comments on PR by codecov.
```

Comment:
remove the `comment:` for now and leave only the comment, that'll be better I guess. Also can you yamllint it? Just activate the bear for it in the coafile for all yml files and run it locally before pushing